src/apps/cardano/get_public_key.py

```diff
@@ -35,8 +35,8 @@ def _get_public_key(root_node, derivation_path: list):
     chain_code = hexlify(node.chain_code()).decode()
     xpub_key = public_key + chain_code
 
-    # In derivation scheme v2 the passphrase is not used
-    root_hd_passphrase = None
+    key = pbkdf2('hmac-sha512', seed.remove_ed25519_prefix(node.public_key()) + node.chain_code(), b'address-hashing', 500).key()
```

Comment:
Is this standardized somehow? If yes, can you please add the standard URL in the code comment?

Reply:
sorry my fault. I removed that commit as the passphrase is really not used in v2. But generation of it is mentioned here: https://cardanodocs.com/technical/protocols/binary-protocols/#address-payload